### PR TITLE
Invitation for community to point their demos/use cases (fixes #356)

### LINKED ISSUE #356: Invitation for community to point to their own demos/use cases
Submitting this on behalf of @anyangocynthia, so she can get a look at our process:

Under tutorials/examples in http://kinto.readthedocs.org/en/latest/tutorials/app-examples.html, we could be more friendly (and provide a little more detail) when asking people to upload their demos e.g. "Here is a complete list of the demos, follow this link to add yours."

### PR DESCRIPTION
**Pull request by leplatrem**
capture d ecran de 2016-01-13 12-08-39

@karlht @anyangocynthia what do you think?

### CODE REVIEW

File: docs/tutorials/app-examples.rst [hunk 1]
```diff
@@ -3,7 +3,12 @@
...
+Usually, the best way to learn is to look at application examples!
 
-* :github:`Complete list <Kinto/kinto/wiki/App-examples>`. It's a Wiki, *add yours!*
+An open Wiki page gives :github:`a complete list of apps in the wild <Kinto/kinto/wiki/App-examples>`!
+
+.. important::
+
+    We would be delighted to highlight demos, use-cases, tutorials,
+    or applications made by the community, so don't hesitate to
+    **add a link to yours**!
```
Comment (Natim) [inline docs/tutorials/app-examples.rst]: we should add a link here to the edit button of the wiki page on github.

Comment (almet) [inline docs/tutorials/app-examples.rst]: +1.

**Comment by almet** [comment]
Great idea otherwise! r+wc

**Comment by anyangocynthia** [comment]
Its Great ! Good job !

**Pull request merged**